### DISCUSSION TITLE
When saving an object via ModelSerializer handle Django ValidationError

### DISCUSSION THREAD

**Discussion by Salaah01** [encode/django-rest-framework#9403] [Ideas & Suggestions]
It is common practice to add validations to the `clean` method in a Django model so that regardless if a user creates/updates a record via the admin page, API or Django shell, the validation will be applied. I consider this to be good practice.

As such, I feel that it would be good for DRF to handle `django.core.exceptions.ValidationError`. At present, it just shows the Django debug error page.

This should be a pretty simple feature, which I'd be more than happy to work on.

**Comment by tonyvg2013** [comment]
Just exactly what you specialize in and I don't presume the knowledge of
clicking the interface of GitHub further from the obvious thing is I have
no reason to even try to script or syntax the same written codeing of
absolutely not One type of single formation of the same developers I'm not
your kind. Absolutely most definitely definitely respect your
classification if that's your expertises. I personally leave that to the
professors of religious continuance of respectful re-editing and discipline
to return to the refrain from and intervention of editorial of a vast Brain
breakers dedication to be exclusive to a very different mindset for my
focus is absolutely a negative refusal to learn even the clicking of the
graphic design of the predators of interference of web design on GitHub is
absolutely not going to be a covitet or even pursue it to be involved in a
relationship that I prefer to never be mutual of many participation for
even Excel is expelled of my memory. I was in elementary and I learn it and
as soon as the grading was over, my only one thing to retrain was the best
way to directly expose the rapid from of pressure in the computer Lab in
elementary School so good riddance Excel for ever to never be remembered
again. Big period! Data entry in the IBM for office secretarial work input
absolutely not a singular time to be timely or pushed to blind type words
for minutes for ever after that six weeks of the grade book was observed to
be completed by my teacher and out of mind temp dump out into no recycling
bin, Trash 🗑️ dose not infer repetition to refuse to trash it. So if you
have extra time to type such dedications to those special people I could
cash app you for your Services.

That's what I'm interested in, have a great rest of this week and for no
reason have no indifference to read your emails but it is absolutely not
understood what you are referencing and to associate it mentally is enert
of my capacity. 💯⭐⭐⭐⭐⭐ Good day 😊
